crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -3484,6 +3484,9 @@ where
                 if self.enabled(Rule::ExprAndFalse) {
                     flake8_simplify::rules::expr_and_false(self, expr);
                 }
+                if self.enabled(Rule::RepeatedEqualityComparisonTarget) {
+                    pylint::rules::repeated_equality_comparison_target(self, expr, *op, values);
+                }
```

Comment:
Nit: You can pass the whole `BoolOp` instead of the individual fields by changing line 3458 to 

```rust
Expr::BoolOp(bool_op @ ast::ExprBoolOp {
	op,
	values,
	range: _
}) => {
```

```suggestion
                if self.enabled(Rule::RepeatedEqualityComparisonTarget) {
                    pylint::rules::repeated_equality_comparison_target(self, bool_op);
                }
```